### PR TITLE
Raise nicer error for floating point instructions

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: lib/fizzy/types.hpp [hunk 1]
```diff
@@ -162,9 +195,63 @@ enum class Instr : uint8_t
...
+    f32_convert_i64_s = 0xb4,
+    f32_convert_i64_u = 0xb5,
+    f32_demote_f64 = 0xb6,
+    f64_convert_i32_s = 0xb7,
+    f64_convert_i32_u = 0xb8,
+    f64_convert_i64_s = 0xb9,
+    f64_convert_i64_u = 0xba,
+    f64_promote_f32 = 0xbb,
+    i32_reinterpret_f32 = 0xbc,
+    i32_reinterpret_f64 = 0xbd,
```
Comment (gumb0) [inline lib/fizzy/types.hpp]:
```suggestion
    i64_reinterpret_f64 = 0xbd,
```

File: lib/fizzy/types.hpp [hunk 2]
```diff
@@ -162,9 +195,63 @@ enum class Instr : uint8_t
...
+    i64_reinterpret_f32 = 0xbe,
```
Comment (gumb0) [inline lib/fizzy/types.hpp]:
```suggestion
    f32_reinterpret_i32 = 0xbe,
```

File: lib/fizzy/types.hpp [hunk 3]
```diff
@@ -162,9 +195,63 @@ enum class Instr : uint8_t
...
+    i64_reinterpret_f64 = 0xbf,
```
Comment (gumb0) [inline lib/fizzy/types.hpp]:
```suggestion
    f64_reinterpret_i64 = 0xbf,
```

File: lib/fizzy/types.hpp (hunk 1 above)
Comment (axic) [reply lib/fizzy/types.hpp]: You're correct:

Screenshot 2020-02-12 at 14 20 25

Comment (axic) [reply lib/fizzy/types.hpp]: Fixed.

**Pull request merged**